packages/nimble-components/src/table/index.ts

```diff
@@ -762,9 +758,14 @@ export class Table<
         this.tableUpdateTracker.trackColumnInstancesChanged();
     }
 
-    private removeActionMenuSlotsFromColumnRequestedSlots(): void {
+    private updateRequestedSlotsForOpeningActionMenu(
+        openActionMenuRecordId: string
+    ): void {
         for (const actionMenuSlot of this.actionMenuSlots) {
-            this.columnRequestedSlots.delete(actionMenuSlot);
+            this.columnRequestedSlots.set(actionMenuSlot, {
+                recordId: openActionMenuRecordId,
+                uniqueSlot: `row-action-menu-${actionMenuSlot}`
```

Comment:
This is kinda now a reserved columnId of `row-action-menu` right?
Should we go down the same uniqueness codepath?

```ts
    const rowActionMenuUniqueId = uniqueId('row-action-menu')
    uniqueSlot: uniquifySlotNameForColumnId(rowActionMenuUniqueId, actionMenuSlot)
```

Reply:
I could update it to go down the same uniqueness code path if we want for consistency, but I don't think it makes a difference in practice. The table's `actionMenuSlots` array has already removed duplicates between columns, so `row-action-menu-${actionMenuSlot}` will never result in duplicate slots created for action menus, and it will also never result in duplicates with column slots that are uniquified to something like `table-column-slot0-${requestedSlot}`.

Thoughts? Do you think I should make the code paths more similar for consistency?

Reply:
I'm sold to keep it as-is!